cluster-provision/k8s/check-cluster-up.sh

```diff
@@ -6,6 +6,9 @@ set -exuo pipefail
 CI=${CI:-"false"}
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 provision_dir="$1"
+SONOBUOY_VERSION=0.19.0
+[ -f "$DIR/$provision_dir/sonobuoy_version" ] && SONOBUOY_VERSION=$(cat "$DIR/$provision_dir/sonobuoy_version" | tr -d '\n')
+export SONOBUOY_VERSION
```

Comment:
fail if there is no version and extract version from conformance.json with `jq .Version`